hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoGetInfoRegression.java

```diff
@@ -94,7 +98,8 @@ private HapiSpec fetchesOnlyALimitedTokenAssociations() {
         final var token6 = "token6";
         final var token7 = "token7";
         final var token8 = "token8";
-        return defaultHapiSpec("FetchesOnlyALimitedTokenAssociations")
+        return propertyPreservingHapiSpec("FetchesOnlyALimitedTokenAssociations")
+                .preserving("tokens.maxRelsPerInfoQuery")
```

Comment:
Why do we need to change this line?

Reply:
Same here. This test "resets" the value of `tokens.maxRelsPerInfoQuery` to 3. We did not notice because `FileUpdateSuite` set the value to 1000 again. Once `FileUpdateSuite` was moved to another group, a third test that required a maximum of 4 failed. 

The clean solution is to use `propertyPreservingHapiSpec` from my understanding, because it really resets the property to its original value.